src/Approvals/SFA.DAS.Approvals.Api/Controllers/CohortController.cs

```diff
@@ -152,25 +152,23 @@ public async Task<IActionResult> GetAddDraftApprenticeshipCourse([FromQuery] lon
         }
 
         [HttpGet]
-        [Route("employer/{accountId}/unapproved/add/confirm-employer")]
         [Route("provider/{providerId}/unapproved/add/confirm-employer")]
-        public async Task<IActionResult> GetHasDeclaredStandards([FromQuery] long? providerId)
+        public async Task<IActionResult> GetConfirmEmployerDeclaredStandards()
```

Comment:
As stated elsewhere, my thought was to just have this called `GetConfirmEmployer`.

Reply:
Done